microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java

```diff
@@ -0,0 +1,112 @@
+package org.jboss.eap.qe.microprofile.jwt.keycloak;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.equalTo;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.concurrent.TimeUnit;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.eap.qe.microprofile.jwt.testapp.Endpoints;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.JaxRsTestApplication;
+import org.jboss.eap.qe.microprofile.jwt.testapp.jaxrs.SecuredJaxRsEndpoint;
+import org.jboss.eap.qe.ts.common.docker.Docker;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+import org.junit.runner.RunWith;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class KeycloakIntegrationHighLevelScenarioTest {
+
+    private static final String KEYCLOAK_REALM_NAME = "foobar";
+
+    private static final String DEFAULT_SERVER_BIND_ADDRESS = "localhost";
```

Comment:
Is it possible to get rid of this constant for example by using something like:
```
@ArquillianResource
protected URL baseApplicationUrl;
```

Reply:
I thought about it and I didn't use it because this is for the Keycloack container purposes only and that thing isn't Aqrquillian managed. Though it would work for sure.

Reply:
thanks, got it.

Reply:
Sorry, re-opening. Could you rename the name of this constant as `DEFAULT_SERVER_BIND_ADDRESS` is usually for WF/EAP server.

Reply:
Thanks! That's a very good point, renaming...

Reply:
Re-pushed